### DISCUSSION TITLE
123云盘提示安全验证需要验证码或者微信登录怎么解决呢

### DISCUSSION THREAD

**Discussion by vpn-web** [AlistGo/alist#6105] [Q&A]
123云盘提示安全验证需要验证码或者微信登录怎么解决呢

**Comment by pongfcnkl** [comment]
用验证码改个密码

**Reply by realSNES** [reply]
没用

**Comment by huolanyun** [comment]
或者 去网站登陆一次就行了，主要是需要验证一下

**Reply by realSNES** [reply]
没用

**Reply by pongfcnkl** [reply]
@realSNES 
如果是在服务器上的，需要服务器搭个代理服务，然后你电脑挂上你服务器的代理在去123改

**Reply by realSNES** [reply]
服务商不让

**Reply by pongfcnkl** [reply]
@realSNES 
不是说一定要挂代理，只要你的电脑能用你服务器的ip中转就行，比如直接起socks5服务
或者不用服务器，直接在本地搭建，除了ios都可以在本地搭建

**Reply by HuanyaoZhang** [reply]
怎么用Linux服务器（非图形化界面）登陆123网盘呀

**Reply by SkyraceMo** [reply]
> @realSNES 不是说一定要挂代理，只要你的电脑能用你服务器的ip中转就行，比如直接起socks5服务 或者不用服务器，直接在本地搭建，除了ios都可以在本地搭建

也没用，刚试了

**Comment by luanshang** [comment]
解决方法：VIP会员可以通过点击“工具中心”->“第三方挂载"，点击”第三方授权管理“添加应用，密码为随机密码，账号为手机号，手机号上方有WebDav地址显示，在Alist的”WebDav“驱动中对应位置填入即可成功添加，然而这个方式与夸克网盘的挂载一样，不能302重定向，故而对服务器带宽有所要求。

**Reply by zealmult** [reply]
> 解决方法：VIP会员可以通过点击“工具中心”->“第三方挂载"，点击”第三方授权管理“添加应用，密码为随机密码，账号为手机号，手机号上方有WebDav地址显示，在Alist的”WebDav“驱动中对应位置填入即可成功添加，然而这个方式与夸克网盘的挂载一样，不能302重定向，故而对服务器带宽有所要求。

服务器带宽才50M 废了

**Reply by SkyraceMo** [reply]
> > 解决方法：VIP会员可以通过点击“工具中心”->“第三方挂载"，点击”第三方授权管理“添加应用，密码为随机密码，账号为手机号，手机号上方有WebDav地址显示，在Alist的”WebDav“驱动中对应位置填入即可成功添加，然而这个方式与夸克网盘的挂载一样，不能302重定向，故而对服务器带宽有所要求。
> 
> 服务器带宽才50M 废了

我也是，搞不了了，转115了

**Comment by luanshang** [comment]
> 123云盘提示安全验证需要验证码或者微信登录怎么解决呢

已解决，使用123的WebDav功能

**Comment by ghost** [comment]
我用爪云（run.claw.cloud）提示我“Failed init storage: 当前账号存在境外登录风险，请使用短信验证码或者微信进行登录。”验证码改密码试了，不行

**Reply by bingoup886** [reply]
> 我用爪云（run.claw.cloud）提示我“Failed init storage: 当前账号存在境外登录风险，请使用短信验证码或者微信进行登录。”验证码改密码试了，不行

一样的问题。。

**Reply by pongfcnkl** [reply]
> 我用爪云（run.claw.cloud）提示我“Failed init storage: 当前账号存在境外登录风险，请使用短信验证码或者微信进行登录。”验证码改密码试了，不行

把文字看完啊，你直接改肯定不行，必须得是爪云的ip

**Reply by ghost** [reply]
感谢

**Comment by zealmult** [comment]
爪云挺垃圾的…感觉不如zeabur
